roles/openshift_node/tasks/upgrade/stop_services.yml

```diff
@@ -1,13 +1,19 @@
 ---
-- name: Stop node and openvswitch services
+- name: Stop node services
   service:
     name: "{{ item }}"
     state: stopped
   with_items:
-  - openvswitch
   - "{{ openshift_service_type }}-node"
   failed_when: false
 
+- name: Stop openvswitch services
```

Comment:
In 3.11 SDN is being run in the pods, so there won't be a service with this name in any case.

This part, however, should be present in `release-3.10` branch.

Reply:
Thanks @vrutkovs  I agree it needs to get to release-3.10

Reply:
This still needs fixing in this PR.

Please create a new PR targetting `release-3.10` with this change